lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -54,6 +54,7 @@ def commands
       "db_nmap"       => "Executes nmap and records the output automatically",
       "db_rebuild_cache" => "Rebuilds the database-stored module cache (deprecated)",
       "analyze"       => "Analyze database information about a specific address or address range",
+      "stats"         => "Show statistics for the database"
```

Comment:
```suggestion
      "db_stats"         => "Show statistics for the database"
```

IMO this feels more future proof/clear to users 🤔

Reply:
agreed.